test/integration_test/common_test.go

```diff
@@ -46,140 +38,8 @@ func createStorageCluster(cluster *corev1alpha1.StorageCluster) error {
 	return err
 }
 
-func uninstallStorageCluster(cluster *corev1alpha1.StorageCluster) error {
```

Comment:
We will have to change basic test to use the methods from test/util now that we have deleted it from here.